api/app/Models/User.php

```diff
@@ -178,6 +178,7 @@ public function getIsProfileCompleteAttribute(): bool
             ) or
             is_null($this->attributes['is_gov_employee']) or
             is_null($this->attributes['has_priority_entitlement']) or
+            ($this->attributes['has_priority_entitlement'] && is_null($this->attributes["priority_number"])) or
```

Comment:
You've added it to `getIsProfileCompleteAttribute()` but how about `scopeIsProfileComplete`? The two seem quite in sync.

Reply:
Huh, was no aware we had both :/ 

Added in [add priority number to profile completion scope](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7313/commits/ca738187731fed287f295ae0788361039ed76031)